source/mongodump.txt

```diff
@@ -816,6 +833,13 @@ Options
    You cannot use the :option:`--archive <mongodump --archive>` option
    with the :option:`--out <mongodump --out>` option.
 
+   .. note::
+
+      :binary:`~bin.mongodump` performance may improve if the process
+      writes to an archive file. For more information, see
```

Comment:
[suggestion] We might want to clarify what information users can find on the page bc, as is, it sounds like I could potentially go to the linked page to learn more about archive files. Maybe we can say something like: 

> For more information on mongodump performance impacts, see...

Reply:
Done!